src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.OpenFileTracker.cs

```diff
@@ -118,7 +118,7 @@ public async static Task<OpenFileTracker> CreateAsync(VisualStudioWorkspaceImpl
                 return new OpenFileTracker(workspace, runningDocumentTable, componentModel);
             }
 
-            private void TryOpeningDocumentsForMoniker(string moniker, ITextBuffer textBuffer, IVsHierarchy hierarchy)
+            private void TryOpeningDocumentsForMoniker(string moniker, ITextBuffer textBuffer, IVsHierarchy? hierarchy)
```

Comment:
What's calling this with a potential null?

Reply:
The tracker could return a null hierarchy if the file isn't open - http://sourceroslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/ProjectSystem/RunningDocumentTableEventTracker.cs,6b90146128b96479